exercises/two-fer/.meta/src/reference/kotlin/Twofer.kt

```diff
@@ -0,0 +1,10 @@
+fun twofer(name: String = ""): String {
```

Comment:
Since we don't need to allow `null` input here, I'd say the closest interpretation of "no name given" would be invoking the `twofer` function with no parameter, a case that is currently not exercised by the tests. To implement that, we should probably update this signature to

```kotlin
fun twofer(name: String = "you"): String
```

and have this exercise focus on a more typical use of default parameter values. Then the first test should mimic https://github.com/exercism/java/blob/master/exercises/two-fer/src/test/java/TwoferTest.java#L16-L22 but with no parameter passed instead of a `null` parameter passed.

The Java track debated what to do with an empty string input, and settled on being very literal about handling it: https://github.com/exercism/java/pull/733. I'd say the same boundary case could be exercised here, since invocation with no parameters is the "right way" to call this function without specifying a name. Any explicit input can therefore be interpreted as something the caller wants to display as the name (even if empty). Does that make sense to fresh eyes?